SalesforceSDK/Salesforce.SDK.Core/Source/Auth/AccountManager.cs

```diff
@@ -61,38 +69,52 @@ public static void DeleteAccount()
         /// <returns></returns>
         public static Account GetAccount()
         {
-            return AuthStorageHelper.GetAuthStorageHelper().RetrieveCurrentAccount();
+            Account account = AuthStorageHelper.GetAuthStorageHelper().RetrieveCurrentAccount();
+            SalesforceApplication.SendToCustomLogger("AccountManager.GetAccount - retrieved current account", LoggingLevel.Verbose);
+            return account;
         }
 
         public static async Task<bool> SwitchToAccount(Account account)
         {
             if (account != null && account.UserId != null)
             {
+                SalesforceApplication.SendToCustomLogger("AccountManager.SwitchToAccount - save pin timer", LoggingLevel.Verbose);
                 PincodeManager.SavePinTimer();
+                SalesforceApplication.SendToCustomLogger("AccountManager.SwitchToAccount - persist credentials", LoggingLevel.Verbose);
                 AuthStorageHelper.GetAuthStorageHelper().PersistCredentials(account);
+                SalesforceApplication.SendToCustomLogger("AccountManager.SwitchToAccount - peek client", LoggingLevel.Verbose);
                 RestClient client = SalesforceApplication.GlobalClientManager.PeekRestClient();
                 if (client != null)
                 {
+                    SalesforceApplication.SendToCustomLogger("AccountManager.SwitchToAccount - clear cookies", LoggingLevel.Verbose);
                     OAuth2.ClearCookies(account.GetLoginOptions());
+                    SalesforceApplication.SendToCustomLogger("AccountManager.SwitchToAccount - call identity service", LoggingLevel.Verbose);
                     IdentityResponse identity = await OAuth2.CallIdentityService(account.IdentityUrl, client);
                     if (identity != null)
                     {
                         account.UserId = identity.UserId;
                         account.UserName = identity.UserName;
                         account.Policy = identity.MobilePolicy;
+                        SalesforceApplication.SendToCustomLogger("AccountManager.SwitchToAccount - persist credentials", LoggingLevel.Verbose);
                         AuthStorageHelper.GetAuthStorageHelper().PersistCredentials(account);
                     }
+                    SalesforceApplication.SendToCustomLogger("AccountManager.SwitchToAccount - refresh cookies", LoggingLevel.Verbose);
```

Comment:
RefreshCookies() ... as commented earlier, if the log point was put into that method, you wouldn't need it again here.